ui-react/packages/atlasmap/src/Atlasmap/useAtlasmapDialogs.tsx

```diff
@@ -197,6 +224,63 @@ export function useAtlasmapDialogs({
     });
   }, [exportAtlasFile, openExportDialog]);
 
+  const onCreateNamespace = useCallback(
+    (docName: string) => {
+      title.current = docName;
+      textVal1.current = "";
+      textVal2.current = "";
+      textVal3.current = "";
+      booleanVal.current = false;
+      openCreateNamespaceDialog(
+        (
```

Comment:
Same here, just pass the function

Reply:
Confused what you mean about this one. I get not needing the useCallback, but I don't get "just pass the function". How and to whom?